packages/perseus/src/widgets/phet-sim.tsx

```diff
@@ -0,0 +1,259 @@
+/* eslint-disable react/forbid-prop-types, react/sort-comp */
+/**
+ * This is a PhET simulation widget. It is used for rendering simulations
+ * from https://phet.colorado.edu/.
+ */
+
+import Banner from "@khanacademy/wonder-blocks-banner";
+import {View} from "@khanacademy/wonder-blocks-core";
+import IconButton from "@khanacademy/wonder-blocks-icon-button";
+import cornersOutIcon from "@phosphor-icons/core/regular/corners-out.svg";
+import {StyleSheet} from "aphrodite";
+import * as React from "react";
+
+import {PerseusI18nContext} from "../components/i18n-context";
+import {getDependencies} from "../dependencies";
+import * as Changeable from "../mixins/changeable";
+import {phoneMargin} from "../styles/constants";
+import {basicBorderColor, borderRadiusLarge} from "../styles/global-constants";
+
+import type {PerseusPhetSimWidgetOptions} from "../perseus-types";
+import type {WidgetExports, WidgetProps} from "../types";
+
+type RenderProps = PerseusPhetSimWidgetOptions; // transform = _.identity
+type Props = WidgetProps<RenderProps, PerseusPhetSimWidgetOptions>;
+
+// For returning user input, but currently the PhET widget
+// does not support accessing user input
+type UserInput = null;
+
+type State = {
+    banner: {
+        message: string;
+        kind: "warning" | "critical";
+    } | null;
+    url: URL | null;
+};
+
+/* This renders the PhET sim */
```

Comment:
Comment: Might be better to be consistent across comment styling, but definitely not a blocker :)